packages/repository/src/repositories/relation.repository.ts

```diff
@@ -26,7 +30,47 @@ export interface HasManyEntityCrudRepository<T extends Entity, ID> {
    * @param options Options for the operation
    * @returns A promise which resolves with the found target instance(s)
    */
-  find(filter?: Filter | undefined, options?: Options): Promise<T[]>;
+  find(filter?: Filter, options?: Options): Promise<T[]>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  update(entity: DataObject<T>, options?: Options): Promise<boolean>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  delete(entity: DataObject<T>, options?: Options): Promise<boolean>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  save(entity: DataObject<T>, options?: Options): Promise<T | null>;
+  /**
+   * 
+   * @param dataObjects 
+   */
+  createAll(dataObjects: DataObject<T>[]): Promise<T[]>;
```

Comment:
AFAIK, juggler does not provide a reliable batch create where all create requests are send in a single request or transaction and where either all or no entities are created. Without that, this tool is just a syntactic sugar for `dataObjects.map(dto => repo.create(dto))`, which adds too little value IMO.

I am proposing to remove `createAll` from LB4 API.